sublime-package.json

```diff
@@ -13,71 +13,95 @@
+                      "default": true,
+                      "description": "[references] code lens.",
+                      "type": "boolean"
                     },
                     "volar.codeLens.scriptSetupTools": {
-                      "type": "boolean",
                       "default": false,
-                      "description": "[ref sugar ☐] code lens."
+                      "description": "[ref sugar \u2610] code lens.",
+                      "type": "boolean"
                     },
-                    "volar.autoWrapParentheses": {
-                      "type": "boolean",
+                    "volar.completion.autoImportComponent": {
                       "default": true,
-                      "description": "Auto-wrap `()` to As Expression in interpolations for fix issue #520."
+                      "description": "Enabled auto-import for component with tag completion.",
+                      "type": "boolean"
                     },
-                    "volar.autoCompleteRefs": {
-                      "type": "boolean",
-                      "default": false,
-                      "description": "Auto-complete Ref value with `.value`."
+                    "volar.completion.preferredAttrNameCase": {
+                      "default": "auto-kebab",
+                      "description": "Preferred attr name case.",
+                      "enum": [
+                        "auto-kebab",
+                        "auto-camel",
+                        "kebab",
+                        "camel"
+                      ],
+                      "enumDescriptions": [
+                        "Auto Detect from Content (Preferred :kebab-case=\"...\")",
+                        "Auto Detect from Content (Preferred :camelCase=\"...\")",
+                        ":kebab-case=\"...\"",
+                        ":camelCase=\"...\""
+                      ],
+                      "type": "string"
                     },
                     "volar.completion.preferredTagNameCase": {
-                      "type": "string",
+                      "default": "auto",
+                      "description": "Preferred tag name case.",
                       "enum": [
+                        "auto",
                         "both",
                         "kebab",
                         "pascal"
                       ],
                       "enumDescriptions": [
+                        "Auto Detect from Content",
                         "<kebab-case> and <PascalCase>",
                         "<kebab-case>",
                         "<PascalCase>"
                       ],
-                      "default": "both",
-                      "description": "Preferred tag name case."
+                      "type": "string"
                     },
-                    "volar.completion.preferredAttrNameCase": {
-                      "type": "string",
+                    "volar.takeOverMode.enabled": {
```

Comment:
Take over mode can be removed.
LSP-volar supports take over mode, but not through this setting.